SPECS-EXTENDED/arpwatch/arpwatch.spec

```diff
@@ -1,11 +1,13 @@
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 %global _vararpwatch %{_localstatedir}/lib/arpwatch
+%global _vargid 777
```

Comment:
Just curious, any specific reason why we choose 777?

Reply:
It was originally 77. Just added a 7 to test that a non-overlapping gid/uid fixes the problem (it does). If there is a better way of doing this or a different range of ids I should use, I'm open to suggestions. 

This package didn't end up being as flexible/helpful as tcpdump, and I don't think anyone is really using it. In that case, I think it is fine to just clean it up a little bit (to the point that it starts up)

Reply:
Probably want to make sure we're compatible with systemd's expectations for UID/GID.
https://systemd.io/UIDS-GIDS/

Tagging @ddstreet for additional suggestions